.env

```diff
@@ -149,8 +149,36 @@ CKANEXT__SAML2AUTH__USER_LASTNAME=last_name
 # Corresponding SAML user field for email
 CKANEXT__SAML2AUTH__USER_EMAIL=email
 
+### saml2 Optional:
+
+# List of email addresses from users that should be created as sysadmins (system administrators)
+# CKANEXT__SAML2AUTH__SYSADMIN_LIST=mail@domain.com
+
+# Indicates that attributes that are not recognized (they are not configured in attribute-mapping),
+# will not be discarded.
+# Default: True
+CKANEXT__SAML2AUTH__ALLOW_UNKNOWN_ATTRIBUTES=true
+
 # Indicates if CKAN should auto create a user for a new SAML login
 # If set False, a new user must be manually created in CKAN before they can log in using SAML
 # If set False, existing user must be in active state to able to log in.
 # Default: True
 CKANEXT__SAML2AUTH__CREATE_USER_VIA_SAML=false
+
+# A list of string values that will be used to set the <NameIDFormat> element of the metadata of an entity.
+# Default: urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
+CKANEXT__SAML2AUTH__SP__NAME_ID_FORMAT=urn:oasis:names:tc:SAML:2.0:nameid-format:persistent urn:oasis:names:tc:SAML:2.0:nameid-format:transient urn:oasis:names:tc:SAML:2.0:nameid-format:emailAddress
```

Comment:
just want to confirm this value as i see `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` in the config but not `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`

Reply:
it is a list of strings, `...nameid-format:persistent`, `...nameid-format:transient`, and more.